src/adapters/aol.js

```diff
@@ -2,17 +2,130 @@ const utils = require('../utils.js');
+  };
+
+  let domReady = (() => {
+    let readyEventFired = false;
+    return fn => {
+      let idempotentFn = () => {
+        if (readyEventFired) {
+          return;
+        }
+        readyEventFired = true;
+        return fn();
+      };
+      let doScrollCheck = () => {
+        if (readyEventFired) {
+          return;
+        }
+        try {
+          document.documentElement.doScroll('left');
+        } catch (e) {
+          setTimeout(doScrollCheck, 1);
+          return;
+        }
+        return idempotentFn();
+      };
+      if (document.readyState === "complete") {
+        return idempotentFn();
+      }
+      if (document.addEventListener) {
+        document.addEventListener("DOMContentLoaded", idempotentFn, false);
+        window.addEventListener("load", idempotentFn, false);
+      } else if (document.attachEvent) {
+        document.attachEvent("onreadystatechange", idempotentFn);
+        window.attachEvent("onload", idempotentFn);
+        let topLevel = false;
+        try {
+          topLevel = window.frameElement === null;
+        } catch (e) {
+        }
+        if (document.documentElement.doScroll && topLevel) {
+          return doScrollCheck();
+        }
+      }
+    };
+  })();
+
+  function dropSyncCookies(pixels) {
+    let pixelElements = parsePixelItems(pixels);
+    renderPixelElements(pixelElements);
+  }
+
+  function parsePixelItems(pixels) {
+    let itemsRegExp = /(img|iframe)[\s\S]*?src\s*=\s*("|')(.*?)\2/gi;
+    let tagNameRegExp = /\w*(?=\s)/;
+    let srcRegExp = /src=("|')(.*?)\1/;
+    let pixelsItems = [];
+
+    if (pixels) {
+      let matchedItems = pixels.match(itemsRegExp);
+      if (matchedItems) {
+        matchedItems.forEach(item => {
+          let tagNameMatches = item.match(tagNameRegExp);
+          let sourcesPathMatches = item.match(srcRegExp);
+          if (tagNameMatches && sourcesPathMatches) {
+            pixelsItems.push({
+              tagName: tagNameMatches[0].toUpperCase(),
+              src: sourcesPathMatches[2]
+            });
+          }
+        });
+      }
+    }
+
+    return pixelsItems;
+  }
+
+  function renderPixelElements(pixelsElements) {
+    pixelsElements.forEach((element) => {
+      switch (element.tagName) {
+        case SYNC_TYPES.img:
+          return renderPixelImage(element);
+        case SYNC_TYPES.iframe:
+          return renderPixelIframe(element);
+      }
+    });
+  }
+
+  function renderPixelImage(pixelsItem) {
+    let image = new Image();
+    image.src = pixelsItem.src;
+  }
+
+  function renderPixelIframe(pixelsItem) {
+    let iframe = document.createElement('iframe');
+    iframe.width = 1;
+    iframe.height = 1;
+    iframe.style.display = 'none';
+    iframe.src = pixelsItem.src;
+    if (document.readyState === 'interactive' ||
+      document.readyState === 'complete') {
+      document.body.appendChild(iframe);
+    } else {
+      domReady(() => {
+        document.body.appendChild(iframe);
```

Comment:
Since you're checking the readyState above, I think removing `domReady` and just using `DOMContentLoaded` should be sufficient here.